src/views/DocumentationTopic.vue

```diff
@@ -27,6 +27,7 @@
             <template #default="slotProps">
               <div class="doc-topic-aside">
                 <QuickNavigationModal
+                  v-if="enableQuickNavigation"
```

Comment:
Shouldn't we update the `enableQuickNavigation` property to check the `isTargetIDE` property in its implementation so that it overrides any config provided in theme-settings?

Reply:
That's what I did initially but I realised that `isTargetIDE` as `true` is already an exclusion for `enableNavigator` and because `StaticContentWidth` component doesn't have `aside` sloth QuickNavigation won't be displayed anyways.

But to really make sure in case the markup changes in the future, I added it back to the computed property https://github.com/apple/swift-docc-render/pull/483/commits/94270127d5dcf183c2872e7ac33cd660cfdf59fd